epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -125,6 +149,18 @@ public TypeName getParameterizedGeneratedName() {
     return parameterizedClassName;
   }
 
+  private void removeMethodIfDuplicatedBySetter(Collection<AttributeInfo> attributeInfos) {
+    for (AttributeInfo attributeInfo : attributeInfos) {
+      Iterator<MethodInfo> iterator = methodsReturningClassType.iterator();
+      while (iterator.hasNext()) {
+        MethodInfo methodInfo = iterator.next();
+        if (methodInfo.name.equals(attributeInfo.getName())) {
```

Comment:
This doesn't check for overloaded methods, which is probably ok for now, but would be nice to have eventually. We would have to check if the `methodInfo` object has just one param, and that the type of that param matches `attributeInfo.type`, before we remove it.